mcp/pacemaker.sysconfig

```diff
@@ -1,5 +1,3 @@
-# For non-systemd based systems, prefix export to each enabled line
```

Comment:
Not sure we've covered all the bases ... systemd, upstart and lsb should be fine when using the provided configs, but I could see some packager using their own, and we don't provide anything for the other init contenders. Perhaps "some systems may require 'export' to be prefixed to each enabled line" -- but we can ignore it for now.